packages/core/contracts/merkle-distributor/implementation/MerkleDistributor.sol

```diff
@@ -71,65 +80,87 @@ contract MerkleDistributor is Ownable {
         address indexed rewardToken,
         address owner
     );
-    event WithdrawRewards(address indexed owner, uint256 amount);
+    event WithdrawRewards(address indexed owner, uint256 amount, address indexed currency);
     event DeleteWindow(uint256 indexed windowIndex, address owner);
 
     /****************************
      *
-     * Admin functions
+     * ADMIN FUNCTIONS
```

Comment:
:)

Reply:
do you also want to remove the extra `*` above and below this to be more consistant with our other styling guides in other contracts? so it would look like this:
```
    /****************************************
     *              ADMIN FUNCTIONS                 *
     ****************************************/

```